index.html

```diff
@@ -288,16 +288,32 @@ <h4>
                     <li><img class="arrow2" src=img/right_arrow.png alt="right_arrow"></li>
                 </ul>
 
-                <img src="img/Separator.png" alt="seperator">
+                <img src="img/Separator.png" id="last_seperator" alt="seperator">
 
         </div>
         </section>
-
-
-
- 
-
-        
+        <section class="contact_form">
+            <h3 class="contact_header">Contact Me</h3>
+            <p class="contact_body">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+                <form class="contact_forms"  action="https://formspree.io/f/mdobrawq" method="POST">
     
+                    <ul class="form_field">
+                        <li class="form_name">
+                            <input type="text" maxlength="5" id="first_name" name="first_name" class="input" placeholder="Name" required>
```

Comment:
- [ ] Kindly, set the maxlength for the Name input to 30 instead of 5 as required.